tensorflow/training/docker/1.15/py3/cu110/Dockerfile.gpu

```diff
@@ -0,0 +1,224 @@
+FROM nvidia/cuda:11.0-base-ubuntu18.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="3"
```

Comment:
why is the major version set to 3 here? not sure i understand, given that the cu100 images are version 7

Reply:
Updated